translator/source_code/cotrip_translator.py

```diff
@@ -0,0 +1,421 @@
+
```

Comment:
Can we organize this file a bit better, it's difficult to follow. For instance we have the usage and what would appear to be the entry point in the middle of the file. Let's move that to the top or bottom to call it out from the rest of the code. Also, there's an if statement floating in the file and not in a method, let's pull that into a "main" method or something similar.

Reply:
Agreed, we should have our "main" method near the top and any functions that the main functions uses, should follow. That way the file can be read "top down"

Reply:
Should be fixed now